js/renderer.ts

```diff
@@ -454,7 +454,6 @@ function applyToYogaNode(
         node.setHeight(h);
         node.setWidth(w);
     } else {
-        applyLayoutToSceneGraph(wrapper, ctx!, true);
```

Comment:
Are you sure about this one? That is quite a wide-ranging change 🤔

Reply:
This was causing the main issue of the UI failing. Ctx turned out to be undefined sometimes. I changed it so there is an `if` around it.